manifests/d/DOSBoxStaging/DOSBoxStaging/0.80.1/DOSBoxStaging.DOSBoxStaging.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.3.4 $debug=NVS1.CRLF.5-1-19041-4123.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DOSBoxStaging.DOSBoxStaging
+PackageVersion: 0.80.1
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/dosbox-staging/dosbox-staging/releases/download/v0.80.1/dosbox-staging-v0.80.1-setup.exe
```

Comment:
Oops, I just found out that there already is a 0.81.0 stable release:

https://github.com/dosbox-staging/dosbox-staging/releases/tag/v0.81.0

Reply:
We will update once this version is published.